src/stateProviders/provideLoggedInUser.js

```diff
@@ -71,8 +62,8 @@ export default provideState({
                 history.push(ROUTES.join);
               } else if (!user.acceptedTerms) {
                 history.push(ROUTES.termsConditions);
-              } else {
-                history.push(ROUTES.login);
+              } else if (['/', '/join', '/orcid'].includes(window.location.pathname)) {
```

Comment:
why should it go to dashboard if you hit /join?

Reply:
it's legacy code (not visible in the left panel for I removed it). I think it means that if you type in /join in the url but you are already logged in then go to dashboard. These pages are hidden if logged in (except for '/')